globus_sdk/base.py

```diff
@@ -102,17 +102,9 @@ def __init__(
         }
 
         # verify SSL? Usually true
-        self._verify = config.get_ssl_verify(self.environment)
+        self._verify_ssl = config.get_ssl_verify()
         # HTTP connection timeout
-        # this is passed verbatim to `requests`, and we therefore technically
-        # support a tuple for connect/read timeouts, but we don't need to
-        # advertise that... Just declare it as an float value
-        if http_timeout is None:
-            http_timeout = config.get_http_timeout(self.environment)
-        self._http_timeout = http_timeout
-        # handle -1 by passing None to requests
-        if self._http_timeout == -1:
-            self._http_timeout = None
+        self._http_timeout = config.get_http_timeout(http_timeout)
```

Comment:
This is what I want the end logic to look like. Maybe the answer is to support `-1` as before, but do the above (removed) handling in the config module.